testonly/setup/flags.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package setup
+
+import (
+	"flag"
+	"testing"
+)
+
+// SetFlag updates a flag value, failing the test if something goes wrong.
+func SetFlag(t *testing.T, name, value string) {
```

Comment:
Do you want to revert the flag after the test is finished?

Reply:
Reverting is done in bulk using `defer flagsaver.Save().Restore()`

Reply:
Thanks